src/components/Button.tsx

```diff
@@ -4,14 +4,19 @@ import classnames from "classnames";
 interface IProps {
   isInverted?: boolean;
   isStretched?: boolean;
+  classname?: string;
+  onClick?: () => void;
 }
 
 export const Button: FC<IProps> = ({
   isInverted = false,
   children,
-  isStretched = false
+  isStretched = false,
+  classname,
+  onClick
 }) => (
   <button
+    onClick={onClick}
```

Comment:
also, I would like to see some examples of `Button` in storybook with the new props